test/e2e/system_reset_test.go

```diff
@@ -89,5 +89,10 @@ var _ = Describe("podman system reset", func() {
 		Expect(session).Should(Exit(0))
 		// default network should exists
 		Expect(session.OutputToStringArray()).To(HaveLen(1))
+
+		session = podmanTest.Podman([]string{"machine", "list", "-q"})
```

Comment:
Was there a machine there earlier?

Reply:
No, machine tests can't be run outside of the machine test suite at the current moment. I can add a TODO here to change it later?

Reply:
Sure, because this test is doing nothing. You could hack it to think there is a machine, and then make sure the machine is removed.